openlibrary/plugins/upstream/merge_authors.py

```diff
@@ -315,17 +315,17 @@ def is_enabled(self):
         return "merge-authors" in web.ctx.features or (user and user.is_admin())
 
     def POST(self):
-        json = web.data()
-        data = simplejson.loads(json)
+        json_data = web.data()
+        data = json.loads(json_data)
```

Comment:
```suggestion
        data = json.loads(web.data())
```
I view variables whose lifespan only lasts until the next line to be an antipattern unless:
1. The lines are really long, and/or
2. The variable name clarifies something that is not obvious.